routes/modelExport.js

```diff
@@ -1,9 +1,33 @@
-var linz = require('../');
+'use strict';
+
+const linz = require('../');
+const setTemplateScripts = require('../lib/scripts');
```

Comment:
This process where you load the methods, then do `Promise.all`... could that all be put into the API? Anything that we do in Linz, I want programmers to be able to do also. At the moment, they can't access this functionality simply. If we add it to the API and we use it, they'll be able to do the same. This will affect a lot of the PR.